code/game/objects/items/devices/seismic_remote.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/device/seismic_remote
```

Comment:
you could probably inherit most of the code from code/game/objects/items/devices/radstorm_remote.dm instead of duplicate